src/rust/engine/fs/src/store.rs

```diff
@@ -1750,6 +1776,10 @@ mod remote {
       }
     }
 
+    ///
+    /// Given a collection of Digests (digests),
```

Comment:
The punctuation in here is odd. More generally, there is a convention in the codebase to use complete sentences in comments (ie ending with a period).